qa/tasks/cephfs/test_volumes.py

```diff
@@ -48,16 +49,20 @@ def __check_clone_state(self, states, clone, clone_group=None,
         args = tuple(args)
 
         count = 0
-        while count < max_count:
-            result = json.loads(self._fs_cmd(*args))
-            current_state = result["status"]["state"]
-            if current_state in states:
-                return
-
-            log.info(f'current clone state = {current_state}')
-            count += 1
-            time.sleep(sleep)
-        self.assertTrue(check < timo)
+
+        msg = (f'Executed cmd "{args}" {max_count} times; clone was never in '
+               f'"{states}" state(s).')
+
+        with safe_while(sleep=sleep, tries=max_count, action=msg) as proceed:
+            while proceed():
+                result = json.loads(self._fs_cmd(*args))
+                current_state = result["status"]["state"]
+                if current_state in states:
+                    return
+
+                log.info(f'current clone state = {current_state}')
+                count += 1
```

Comment:
Where is `count` used here?

Reply:
Not needed since we are using `safe_while`; fixed.